src/FormBuilder.php

```diff
@@ -71,15 +76,28 @@ public function __construct( $method, $signatureKeyId, $signatureSecret, $accoun
     /**
      * Get parameters for add card request
      *
-     * @param bool $accept_cvc_required
+     * @param bool $acceptCvcRequired
+     * @param bool $skipFormNotifications
+     * @param bool $exitIframeOnResult
+     * @param bool $exitIframeOn3ds
+     * @param bool $use3ds
      * @return Form
      */
-    public function generateAddCardParameters( $accept_cvc_required = false )
+    public function generateAddCardParameters( $acceptCvcRequired = false, $skipFormNotifications = false,
+                                               $exitIframeOnResult = false, $exitIframeOn3ds = false, $use3ds = false )
     {
         $commonParameters = $this->createFormParameterArray();
 
-        if($accept_cvc_required)
+        if($acceptCvcRequired)
             $commonParameters[self::$SPH_ACCEPT_CVC_REQUIRED] = true;
+        if($skipFormNotifications)
+            $commonParameters[self::$SPH_SKIP_FORM_NOTIFICATIONS] = true;
+        if($exitIframeOnResult)
+            $commonParameters[self::$SPH_EXIT_IFRAME_ON_RESULT] = true;
+        if($exitIframeOn3ds)
+            $commonParameters[self::$SPH_EXIT_IFRAME_ON_THREE_D_SECURE] = true;
+        if($use3ds)
```

Comment:
Given the nature of three-value-boolean, this should check whether or not it is null, or some other approach